packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -292,24 +303,39 @@ contract DeployImplementations is Script {
         });
     }
 
+    // Deploy and initialize the OPStackManager contract that is fronted by a proxy.
     function createOPSMContract(
         DeployImplementationsInput _dii,
         DeployImplementationsOutput,
-        OPStackManager.Blueprints memory blueprints
+        OPStackManager.Blueprints memory blueprints,
+        string memory release,
+        OPStackManager.ImplementationSetter[] memory setters
     )
         internal
         virtual
-        returns (OPStackManager opsm_)
+        returns (OPStackManager opsmProxy_)
     {
         SuperchainConfig superchainConfigProxy = _dii.superchainConfigProxy();
         ProtocolVersions protocolVersionsProxy = _dii.protocolVersionsProxy();
+        ProxyAdmin proxyAdmin = _dii.superchainProxyAdmin();
 
-        vm.broadcast(msg.sender);
-        opsm_ = new OPStackManager({
-            _superchainConfig: superchainConfigProxy,
-            _protocolVersions: protocolVersionsProxy,
-            _blueprints: blueprints
-        });
+        // We broadcast as the proxyAdminOwner address due to the ProxyAdmin's `onlyOwner` modifier.
```

Comment:
incorrect comment? looks like a regular broadcast

Reply:
True, forgot to remove this after we decided to use `msg.sender`. Updated: https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23#diff-9076b9402a282fc4bfeb1038536a8aec45ae7603f734c3f23fea747caf85f53fL322